src/plugins/home/public/plugin.ts

```diff
@@ -166,6 +166,7 @@ export interface HomePublicPluginSetup {
    * The environment service is only available for a transition period and will
    * be replaced by display specific extension points.
    * @deprecated
+   * @removeBy 8.8.0
    */
   environment: EnvironmentSetup;
```

Comment:
This is flagged as deprecated while being actively used with no alternatives... Are we sure about this?

Reply:
Looking through the code, it's only used in 2 places: `cloud` and `apm`. And APM also registers the featureCatalog:

1. `isCloud: true` is the only one used inside the `home` to change anything
2. `apmUi: true` doesn't seem to be used anywhere else